### DISCUSSION TITLE
opentrack 2022.1.1

### DISCUSSION THREAD

**Discussion by sthalik** [opentrack/opentrack#1419] [Announcements]
### Fixes

- don't show embedded module dialogs on low resolutions; override by settings the `OPENTRACK_MODULE_TABS` environment variable to 0 or 1. issue #1407; reported by @00greena.
- fix handling of the Escape key with embedded module dialogs. as a consequence of this, the Escape key will now cancel and close the options dialog. report this as issue or in discussion thread if you hate this.

### Features

- allow mouse shortcuts for extended buttons (not left, right, middle or mousewheel). keyboard modifiers like Ctrl or Shift are ignored for mouse shortcuts. issue #583; requested by many people.
- support multiple copies of the same camera model being connected at the same time. as a consequence of this, you may need to reselect the camera inside the tracker's options, but a migration has been added and should work in most cases. issue #1415; reported, tested and with additional help by @hrafnkelle and @nuxil.

<hr /><em>This discussion was created from the release <a href='https://github.com/opentrack/opentrack/releases/tag/opentrack-2022.1.1'>opentrack 2022.1.1</a>.</em>

**Comment by rvkshn-10** [comment]
Which file in the assets do I install?

**Reply by sthalik** [reply]
The installer is `opentrack-2022.1.1-win32-setup.exe`.

**Comment by sutex** [comment]
Any updates when this will get fixed/implemented - Issue Virtual keys F13 -F24 #1012 does not work #1269

**Comment by terry911204** [comment]
Um so dum question here
Is there a way to change the opentrack window size?
I have a 14-inch screen and couldn't see the apply/ok button in the options tab

**Reply by sthalik** [reply]
What's your display resolution then?

**Comment by terry911204** [comment]
It’s 1920*1080

**Reply by sthalik** [reply]
Are you using display scaling in Windows?

Anyway, set the `OPENTRACK_MODULE_TABS` environment variable to 0.

**Comment by Vatharian** [comment]
I have an issue with vjoystick. OpenTrack doesn't recognize that it is installed - when clicking configure button it shows the message to go to jvoystick site (which is dead) or download directly. Is there any trick to enable it?

Windows 10 21H2, vJoy 2.1.9.1-160719.

**Comment by GO63-samara** [comment]
@Vatharian 
You need to install VJoySetup.exe:
https://sourceforge.net/projects/vjoystick/files/latest/download

**Reply by Vatharian** [reply]
I DO have it installed, and it is not detected. I did complete the reboot that vJoy requires, I have added second joystick, the Feeder app works, I even tried running openTrack as Adminstrator - it simply doesn't see and displays the message still.

**Reply by GO63-samara** [reply]
The message is always displayed, this is normal.
"Configure VJoy" can be launched from the Start button menu.
It is convenient to check the work in Pointy's Joystick Test:
http://www.planetpointy.co.uk/joystick-test-application/
![image](https://user-images.githubusercontent.com/63638386/160302020-6b20f94d-d3c9-407d-b796-5c9c9c361c36.png)

**Comment by Torquesplit** [comment]
Help,  Have been trying to download OpenTrack onto my iMac running OS Big Sur with no luck. Eventually got a Folder into my Download Section but can't figure how to open it as an Application. As a non Computer user I cannot find any download instructions that mean anything to me! Am I missing a simple step or is there an issue with OpenTrack and Mac OS?

**Reply by sthalik** [reply]
OSX isn't supported as well as Windows and Linux, and for this reason no binaries are being provided. Existing developers don't use any Apple hardware. Right now OSX support is only useful for people willing to jump through *many* hoops, and live with broken features.
